doc/ref_cert/README.md

```diff
@@ -1,9 +1,136 @@
 [<< Back](../)
 
-# CNTT Reference Conformance
+# CNTT Reference Conformance (RC) Test Case Integration
 
-<a name="available-cr"></a>
+All CNTT conformance suites must fulfill the following test case integration
+requirements to guarantee a smooth overall integration, the same end user
+actions and a unique test result format (e.g. OPNFV test result database)
+needed by the end users and the test case result verification programs (e.g.
+[OVP](https://www.opnfv.org/verification)). Historically, these rules were
+agreed by RC1 team and have been applied since
+[CNTT Snezka](https://github.com/cntt-n/CNTT/wiki/Snezka). The new RC2 stream
+asks to move them in this common place applicable to all CNTT conformance
+suites.
+
+<a name="ri-rc-toolchaings"></a>
+## CNTT RI and RC toolchains
+
+[OPNFV](https://www.opnfv.org/) has built a complete CI/CD toolchain
+for continuously deploying and testing cloud infrastructure.
+
+As for all OPNFV installer projects,
+[Jenkins](https://build.opnfv.org/ci/view/cntt/) triggers scenario deployments,
+runs the OPNFV gating test cases and then publishes all test results in the
+[centralized test database](https://docs.opnfv.org/en/stable-hunter/_images/OPNFV_testing_working_group.png)
+and all artifacts (reports, logs, etc.) to
+[an S3 compatible storage service](http://artifacts.opnfv.org/).
+
+The CNTT verification, validation and conformance processes will leverage
```

Comment:
```suggestion
The CNTT verification, validation, and conformance processes will leverage
```

Reply:
it doesn't seem right in my native language. Can someone second this proposal?

Reply:
It's fine.  It's called the Oxford Comma and brings with it much debate about where it should and shouldn't be used.  Both options here are grammatically correct, but adding the comma can make it clearer.

Reply:
@tomkivlin Maybe we should get an agreement to follow the Oxford comma throughout for consistency. FYI in North America it is frowned upon and hence have lost the habit.

Reply:
It should be used throughout for clarity. https://annhandley.com/oxford-comma/ check this if you want to understand why and have a laugh